changelog_unreleased/angular/16474.md

```diff
@@ -0,0 +1,17 @@
+#### Support Angular `@let` declaration syntax (#16474 by @sosukesuzuki)
```

Comment:
```suggestion
#### Support `@let` declaration syntax (#16474 by @sosukesuzuki)
```

The changelog already in a language directory

Reply:
https://github.com/prettier/prettier/pull/16474/commits/26ba5724767e9940d36564c572cc38f12a7b3ad0